README.md

```diff
@@ -12,8 +12,8 @@ This is based on https://github.com/ryanlovett/nbnovnc.
 
 When this extension is launched it will run a Linux desktop on the Jupyter single-user server, and proxy it to your browser using VNC via Jupyter.
 
-If a `vncserver` executable is found in `PATH` it will be used, otherwise a bundled TightVNC server is run.
-You can use this to install vncserver with support for other features, for example the [`Dockerfile`](./Dockerfile) in this repository installs TurboVNC for improved OpenGL support.
+This extension requires a VNC server to be installed on the system, along with a desktop or equivalent.
```

Comment:
Given we have code [like this](https://github.com/jupyterhub/jupyter-remote-desktop-proxy/blob/a0f7620952a32b77de80ba99d6bbdb34d6b27c5b/jupyter_remote_desktop_proxy/setup_websockify.py#L20) I'd say we only support tigervnc and turbovnc right now. We should explicitly mention that, and say that other VNC servers *may* work if they are available on the terminal as a `vncserver` executable.